stdlib/camlinternalFormat.ml

```diff
@@ -438,11 +440,12 @@ let bprint_altint_fmt buf ign_flag iconv pad prec c =
 
 (***)
 
-(* Print the optional '+' associated to a float conversion. *)
-let bprint_fconv_flag buf fconv = match fst fconv with
+(* Print the optional '+', ' ' and/or '#' associated to a float conversion. *)
+let bprint_fconv_flag buf fconv = begin match fst fconv with
```

Comment:
I would break the line before `begin` to clarify the two-steps structure.

Reply:
Right, done